.gitignore

```diff
@@ -1,5 +1,5 @@
 *.py[cod]
-.*
+.env
```

Comment:
What is `.env` and why is it special? We should be ingoring all dot-files with some exceptions. I don't want to have to list all possible IDE/environment cruft here.